pkg/cover/templates/cover.html

```diff
@@ -94,13 +94,31 @@
     .active {
       display: block;
     }
+    .total-left {
+      padding-left: 16px;
+    }
+    .total {
```

Comment:
Is `.total` used anywhere?

Reply:
Nice catch, I forgot to change the class below!